src/components/Session/DateCard.jsx

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Breaking things up into modules is great when things are used in many places. But when making something for the first time, just keep it simple in just one file.

Having so many types of cards is confusing especially since it's not actually a UI element.

Additionally, loading shouldn't be done by individual components. Instead, use https://github.com/irisli/stellarterm/blob/master/src/components/Loading.jsx